src/_gettsim/transfers/kinderzuschl/kost_unterk.py

```diff
@@ -21,71 +20,131 @@ def kinderzuschl_kost_unterk_m_bg(
     -------
 
     """
-    warmmiete_m_hh = bruttokaltmiete_m_hh + heizkosten_m_hh
-    anteil_warmmiete_m_bg = warmmiete_m_hh * _anteil_personen_in_haushalt_bg
 
-    out = _kinderzuschl_wohnbedarf_eltern_anteil_bg * anteil_warmmiete_m_bg
+    return (
+        bruttokaltmiete_m_bg + heizkosten_m_bg
+    ) * _kinderzuschl_wohnbedarf_eltern_anteil_bg
 
-    return out
 
-
-def bruttokaltmiete_m_bg(
+def bruttokaltmiete_m_bg(  # noqa: PLR0913
+    anz_paare_hh: int,
     bruttokaltmiete_m_hh: float,
-    _anteil_personen_in_haushalt_bg: float,
+    anz_kinder_bis_17_bg: int,
+    anz_kinder_bis_17_hh: int,
+    anz_erwachsene_bg: int,
+    anz_erwachsene_hh: int,
+    kinderzuschl_params: dict,
 ) -> float:
-    """Share of household's monthly rent attributed to the Bedarfsgemeinschaft.
+    """Monthly rent attributed to the Bedarfsgemeinschaft.
```

Comment:
Do we have any legal reference that it is done that way? Should reference it here.